src/compile/projection/parse.ts

```diff
@@ -66,25 +75,25 @@ function dryMerge(first: ProjectionComponent, second: ProjectionComponent): Proj
     // both have property and an equal value for property
     if (first.explicit.hasOwnProperty(prop) &&
       second.explicit.hasOwnProperty(prop) &&
+      // some properties might be signals or objects and require hashing for comparison
       hash(first.get(prop)) === hash(second.get(prop))) {
```

Comment:
If we want exact comparison, we should use stringify here more than hash?

Reply:
hash is stringify.